src/Redux/configureStore.js

```diff
@@ -0,0 +1,16 @@
+import { applyMiddleware, combineReducers, configureStore } from '@reduxjs/toolkit';
```

Comment:
Great job adding combinedReducer. Although it would work better if you import it from redux and not '@reduxtoolkit'